src/drivers/PortFolder/WDMTestingTemplate/driver/sdv-user.sdv

```diff
@@ -0,0 +1,5 @@
+CancelSpinLock

```

Comment:
Do we need references to SDV? If not, please rename.

Reply:
I removed it.